### ISSUE TITLE
Composer

### ISSUE THREAD

**Issue by makbays** [jurajmajer/enrico#24]
Why don't we publish this on Packagist and make able to get by Composer?

**Comment by jurajmajer** [comment]
Good point. I will take a look at Packagist documentation (https://packagist.org/) on how to publish the package.

**Comment by jurajmajer** [comment]
I have added composer.json to enrico project and submit the project to packagist.org - https://packagist.org/packages/kayaposoft/enrico. Feel free to report any issues or improvements.
